cicd/jenkins/images-other-jenkins.adoc

```diff
@@ -18,7 +18,7 @@ For example:
 
 [source,terminal]
 ----
-$ podman pull registry.redhat.io/openshift4/ose-jenkins:<v4.3.0>
+$ podman pull registry.redhat.io/ocp-tools-4/ose-jenkins:<image_tag>
```

Comment:
you also need a placeholder for the image_name which changed: it is now `jenkins-rhel8`

https://catalog.redhat.com/software/containers/ocp-tools-4/jenkins-rhel8/